source/fundamentals/connection/tls.txt

```diff
@@ -0,0 +1,317 @@
+.. _node-connect-tls:
+
+==========================
+Enable TLS on a Connection
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect to MongoDB instances with the
+`TLS/SSL <https://en.wikipedia.org/wiki/Transport_Layer_Security>`__
+security protocol. To configure your connection to use TLS/SSL, enable
+the TLS/SSL settings in either your connection string or the
+`MongoClientOptions <{+api+}/interfaces/MongoClientOptions.html>`__
+for your client.
+
+.. important:: TLS vs. SSL
+   
+   TLS is the successor protocol to SSL, but the terms are often used
+   interchangeably. Since TLS is an improved version of SSL, this guide
+   describes how to connect to MongoDB with the TLS protocol.
+
+.. _node-enable-tls:
+
+Enable TLS
+----------
+
+You can enable TLS on a connection to your MongoDB instance
+in the following ways:
+
+- Setting the ``tls`` option to ``true`` in your ``MongoClientOptions``
+- Setting the ``tls`` option to ``true`` in your connection string
+
+.. tabs::
+
+   .. tab:: MongoClientOptions
+      :tabid: mongoclientoptions
+
+      To enable TLS on a connection by using an option in a
+      ``MongoClientOptions`` instance, set the property ``tls`` as
+      ``true`` and pass the options to the ``MongoClient`` constructor:
+      
+      .. code-block:: js
+         :emphasize-lines: 2
+
+         var myClientSettings = {
+           tls: true,
+           ...
+         };
+         
+         const client = new MongoClient(uri, myClientSettings);
+
+   .. tab:: Connection String
+      :tabid: connection string
+
+      To enable TLS on a connection by using a connection string option,
+      set the parameter ``tls`` as ``true`` in the connection string URI
+      that you pass to the ``MongoClient`` constructor:
+
+      .. code-block:: js
+         :emphasize-lines: 1
+
+         const uri = "mongodb://<hostname>:<port>?tls=true";
+         const client = new MongoClient(uri, myClientSettings);
+
+.. _node-configure-tls-certificates:
+
+Configure Certificates
+----------------------
+
+Applications that initiate TLS requests require access to
+cryptographic certificates that prove the identity of the application
+itself and other applications that the application
+interacts with. To connect to MongoDB with TLS, your certificates must
+be stored as ``PEM`` files, which are concatenated certificate
+containers.
```

Comment:
wiki article seems like a good addition

Reply:
Since this is going to an external site, can we make it clear per [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear)?

Even removing the inline link and putting it in an admonition instead.